example/ios/Podfile.lock

```diff
@@ -703,4 +709,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 446419b91f3d523996e9ca2fa51242b50bd76552
 
-COCOAPODS: 1.11.3
+COCOAPODS: 1.12.0
```

Comment:
I can get behind this, but we all kind of need to switch together.  Alternatively, we could back this and the Gemfile.lock out into a separate PR for clarity.  Both good ideas - just need to do this across all devs & repos.

Reply:
I don't remember explicitly upgrading to `1.12.0`. I downgraded back to `1.11.3` until we get the chance to discuss as a team.